src/components/Item/VisualizationItem/ItemContextMenu/__tests__/ItemContextMenu.spec.js

```diff
@@ -47,17 +47,28 @@ test('renders just the button when menu closed', () => {
 })
 
 test('renders exit fullscreen button', () => {
-    useSystemSettings.mockImplementationOnce(() => mockSystemSettingsDefault)
-    const props = Object.assign({}, defaultProps, {
-        isFullscreen: true,
-    })
-    const { container } = render(
+    useSystemSettings.mockImplementation(() => mockSystemSettingsDefault)
+    const gridItemClassName = `reactgriditem-${defaultProps.item.id}`
```

Comment:
```suggestion
    const gridItemClassName = getGridItemDomElementClassName(defaultProps.item.id)
```

Reply:
Never mind, didn't realise this was the test code 🙈

Reply:
Did it anyway.